tests/Issues/IssueDowngradeUnionTypeAndParameterTypeWideningWithConfiguration/config/configured_rule.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\DowngradePhp80\Rector\FunctionLike\DowngradeUnionTypeDeclarationRector;
+use Rector\DowngradePhp72\Rector\ClassMethod\DowngradeParameterTypeWideningRector;
+use Rector\Tests\Issues\IssueDowngradeUnionTypeAndParameterTypeWideningWithConfiguration\Source\WithInstanceManagerServiceTrait;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->import(__DIR__ . '/../../../../config/config.php');
+
+    $rectorConfig->rule(DowngradeUnionTypeDeclarationRector::class);
+    $rectorConfig->rule(DowngradeParameterTypeWideningRector::class);
+    
+    $rectorConfig
+        ->ruleWithConfiguration(DowngradeParameterTypeWideningRector::class, [
```

Comment:
@leoloso I think the easiest way to fix it is by remove from existing set list, then apply yours, something like this:

```suggestion
    $rectorConfig->services()->remove(DowngradeParameterTypeWideningRector::class);

    $rectorConfig
        ->ruleWithConfiguration(DowngradeParameterTypeWideningRector::class, [
            WithInstanceManagerServiceTrait::class => ['getInstanceManager'],
        ]);
```

By that, you will get the following output:

```diff
 class ParentClass
 {
-    public function import(bool $ignoreErrors = false, string $sourceResource = null, string|array $exclude = null)
+    /**
+     * @param string|mixed[] $exclude
+     */
+    public function import(bool $ignoreErrors = false, string $sourceResource = null, $exclude = null)
     {
     }
 }
@@ @@
 
 class ChildClass extends ParentClass
 {
-    public function import(bool|string $ignoreErrors = false, string $sourceResource = null, $exclude = null): mixed
+    /**
+     * @param bool|string $ignoreErrors
+     */
+    public function import($ignoreErrors = false, string $sourceResource = null, $exclude = null): mixed
     {
     }
 }
```

which should be corret, see https://3v4l.org/QZkl1#v7.2.34

Reply:
@leoloso Oh, I see, it seems the issue will be on php 7.1, 

ref https://getrector.com/demo/b00b3ca4-b28c-428f-a166-3071faa9cbde
ref https://3v4l.org/jjeZc#v7.1.32

Reply:
@leoloso I think the solution is define both `import` and `getInstanceManager` to the config, see 

- https://getrector.com/demo/46da785a-9c8e-4ea0-a14b-483ef16cf5da
- https://3v4l.org/bdEcj#v7.1.32

That will work on php 7.1.